src/axolotl/utils/config.py

```diff
@@ -4,6 +4,7 @@
 import os
 
 import torch
+from transformers.utils import is_torch_bf16_gpu_available
```

Comment:
`torch` has a similar function, `torch.cuda.is_bf16_supported()`. Would it be better to use it directly through torch instead of transformers?

Reply:
thanks, updated the check to use that.

Reply:
actually, using the torch native check breaks the unit tests since cuda isn't available. transformers utils has a cuda check built in.